examples/commoncrawl/src/main/ml-config/database-fields/final-database.xml

```diff
@@ -0,0 +1,125 @@
+<database-properties xmlns="http://marklogic.com/manage">
```

Comment:
We want to gitignore src/main/ml-config as well, unless we get to a point where we need to add a non-generated file to the directory